### ISSUE TITLE
Displaying colour name whilst using swatch could/should be native?

### ISSUE THREAD

**Issue by M-H1** [bigcommerce/cornerstone#1847]
Colour name appears on desktop only when hovering over colour. No colour name displayed on tablet or mobile.

New JavaScript coding could be added to display colour name on every device when clicking/tapping swatch colour - next to 'Colour:' instead of 'required' or similar placement.

**Comment by mattcoy-arcticleaf** [comment]
If this gains traction, I am willing to write a few lines of code to make this an option in Theme Settings.

**Comment by jake1490** [comment]
I also wish this to be an option. My store sells camouflage apparel and the only way customers can be sure they're selecting the correct pattern is by seeing a visible name of the swatch. A lot of business is gained on customers ordering multiple products of the same pattern.

**Comment by flinnwildlife** [comment]
I'm not sure how this isn't a native option. The ability to display the name with the swatch is critical for customer transparency.

**Comment by wschrank** [comment]
Should absolutely be a native function with consumers on mobile.

**Comment by itsjeffreyab97** [comment]
I agree with the three comments above. BigCommerce should integrate the colors names into the gallery as much as possible on mobile. tablet and desktop. This seem to be a standard feature on other store but not these.

**Comment by mattcoy-arcticleaf** [comment]
No comments in 19 days, then 4 in a matter of 20 minutes? Was there some sort of forum or group post linking to this? 

Anyway, I'll go ahead and submit a Pull Request when I get a chance, though it may be some time now since we are currently quite busy.

**Comment by mattcoy-arcticleaf** [comment]
Sorry I didn't get around to this in a timely fashion. It appears that this has been resolved here: https://github.com/bigcommerce/cornerstone/pull/1976
